NuGet.config

```diff
@@ -2,6 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="sqlclient" value="https://sqlclientdrivers.pkgs.visualstudio.com/public/_packaging/sqlclient/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Need the new packages in the sqlclient feed or a new feed

Reply:
The sqlclientdrivers feed upstreams from NuGet.org. If you revert this and push, I believe the CI will have permissions to pull from upstream and install the new versions to the feed.

Reply:
@David-Engel It did not work for me locally, but maybe things have changed?

Reply:
Got it! I will revert